### ISSUE TITLE
UI

### ISSUE THREAD

**Issue by nishant23122000** [infiniteoverflow/Flutter-Tutorial-App#22]
Can i work on this both Screen because i already created this type of screen in my recent flutter project?

![Screenshot 2021-02-17 151640](https://user-images.githubusercontent.com/53515714/108186334-42a03b00-7133-11eb-8cef-803bf5e7a4de.png)

**Comment by infiniteoverflow** [comment]
Hii @nishant23122000 , actually this was a tentative ui i had given for the app , but we can create this ui only after we setup the modules for Dart and Flutter in our app. So this ui might need sometime before we start its implementation.

Once we reach that point , i will open up a new issue for the same. For now i will be closing this.
